src/styles/components/components/SSIEntityIcon/index.ts

```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components/native'
+
+export const SSIEntityIconStyled = styled.View`
+ margin-top: 5px;
+ align-items: center;
+ justify-content: center;
+ aspect-ratio: 1;
+ border-radius: 17px;
```

Comment:
what happends with this when i set the size of the icon to, lets say 50? will it still be round? My guess is, it wont :). for now i think we can set a fixed size for the profile icon. so move the width to this style component

Reply:
moved to the styles for now. The dynamic styling for all sizes has not been added for now.